app/main.py

```diff
@@ -1,3 +1,7 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    count = 0
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count += 1
+
```

Comment:
There is an indentation issue here. The line should be indented to align with the rest of the code inside the function. This will ensure that the function returns the count correctly after the loop completes.